### PR TITLE
Ginkgo: Added example test on Nightly using public image

### PR DESCRIPTION
**Pull request by eloycoto**
Hi, 

This PR adds the test to check the Kubernetes example in the nightly builds.

It reads the example (YAML), changes the values of the config map and imports the manifest. It is using the public image, instead of the local one. 

Regards.

### CODE REVIEW

File: test/helpers/utils.go [hunk 1]
```diff
@@ -180,3 +182,26 @@ func reportMap(path string, reportCmds map[string]string, node *SSHMeta) {
 		}
 	}
 }
+
+// DecodeYAMLOrJSON reads a json or yaml file and export all documents as an
```
Comment (ianvernon) [inline test/helpers/utils.go]: export --> exports

File: test/helpers/utils.go [hunk 2]
```diff
@@ -180,3 +182,26 @@ func reportMap(path string, reportCmds map[string]string, node *SSHMeta) {
...
+// array of interfaces. In case of a invalid path or invalid format it returns
```
Comment (ianvernon) [inline test/helpers/utils.go]: a invalid path --> an invalid path

File: test/k8sT/Nightly.go [hunk 3]
```diff
@@ -159,3 +167,109 @@ var _ = Describe("NightlyK8sEpsMeasurement", func() {
...
+		for _, v := range apps {
+			res, err := kubectl.GetPodNames(helpers.DefaultNamespace, fmt.Sprintf("id=%s", v))
+			Expect(err).Should(BeNil())
+			appPods[v] = res[0]
+			logger.Infof("PolicyRulesTest: pod=%q assigned to %q", res[0], v)
+		}
+		return appPods
+	}
+
+	It("Check k8s Example is working correctly", func() {
```
Comment (ianvernon) [inline test/k8sT/Nightly.go]: k8s --> K8s

**Comment by ianvernon** [comment]
This is a great idea, as it will ensure that our example YAML is valid. Thanks for doing this! :)

File: test/k8sT/Nightly.go [hunk 4]
```diff
@@ -159,3 +167,109 @@ var _ = Describe("NightlyK8sEpsMeasurement", func() {
...
+	BeforeEach(func() {
+		initialize()
+	})
+
+	AfterEach(func() {
+		kubectl.Delete(demoPath)
+		kubectl.Delete(l3Policy)
+	})
+
+	getAppPods := func() map[string]string {
```
Comment (ianvernon) [inline test/k8sT/Nightly.go]: Can you add a comment saying what the values of these strings represent? I.e., maps [x] to [y]

**Review by ianvernon — CHANGES_REQUESTED** [review]
Some minor nitpicks. This is a great change as it ensures that the files used by the public work as intended.

**Review by tgraf — APPROVED** [review]

**Pull request merged**